wgengine/magicsock/magicsock.go

```diff
@@ -96,6 +97,9 @@ func useDerpRoute() bool {
 	return false
 }
 
+// pad4on32bit is 4 bytes on 32-bit, 0 bytes on 64-bit.
+type pad4on32bit [4 * (1 - ((^uint(0))>>32)&1)]byte
```

Comment:
oof, ETOOMUCHMAGIC. Can you make a types/pad32 package or something to hide it? I presume we're going to need this in other places.